cmd/root.go

```diff
@@ -441,6 +442,9 @@ func constructBaseObj(paths []string) string {
 		}
 
 		name := strings.TrimSuffix(path.Base(p), ext)
+		if !utils.IsASCIIIdentifier(name) {
+			name = fmt.Sprintf(`"%s"`, name)
```

Comment:
Tiny nit, isn't this just `sanitizeComponent`?

Reply:
yes, but `sanitizeComponent` isn't visible outside of params package ... do you think it should be public?

Reply:
I'd make this public, just so the name munging logic is in one place. Then come back and clean this up when we move `constructBaseObj` somewhere else.